MobileCenter/MobileCenter.xcodeproj/project.pbxproj

```diff
@@ -1054,6 +1051,7 @@
 				04BED70B1ECB636900E20975 /* OCHamcrest */,
 				6E171B121D22FC2E000DC480 /* OCMock */,
 				6E171B221D22FF40000DC480 /* OHHTTPStubs */,
+				6E171ADB1D22F6ED000DC480 /* OCHamcrestIOS.framework */,
```

Comment:
Remove this one. OCHamcrest folder contains all frameworks. Make sure the project is referencing a right file in `Build Phase`.